pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -294,4 +302,8 @@ private double getTotalNicUsageTxKb(List<String> nics) {
     private static long readLongFromFile(String path) throws IOException {
         return Long.parseLong(new String(Files.readAllBytes(Paths.get(path)), Charsets.UTF_8).trim());
     }
+
+    private static double readDoubleFromPath(Path path) throws IOException {
+        return Double.parseDouble(new String(Files.readAllBytes(path)));
```

Comment:
Double.parseDouble may throw a IllegalArgumentException (NumberFormatExcepton)
we should wrap it in a IOException